components/MapView.js

```diff
@@ -404,6 +425,14 @@ class MapView extends React.Component {
     this._runCommand('takeSnapshot', [width, height, finalRegion, callback]);
   }
 
+  _uiManagerCommand(name) {
+    return NativeModules.UIManager[airMapName(this.props.mapProvider)].Commands[name];
```

Comment:
It seems these two methods are duplicated from `common.js`.  Perhaps pull them out to functions and re-use.